build.gradle

```diff
@@ -184,49 +171,14 @@ task createVersion(type: Copy) {
 // the createVersion task should be called before compileJava or the version service will not work
 compileJava.dependsOn createVersion
 
-def imageName = "nexus.iex.ec/iexec-core"
-
-project.ext.getDockerImageNameFull = {
-    def imageNameWithVersion = imageName + ":${version}"
-    if (isSnapshotVersion) {
-        return imageNameWithVersion + "-" + gitShortCommit
-    } else {
-        return imageNameWithVersion
-    }
-}
-
-project.ext.getDockerImageNameShortCommit = {
-    return imageName + ":" + gitShortCommit
-}
-
-task prepareDockerFile(type: Copy) {
-    delete 'build/main/resources/'
-    from 'Dockerfile'
-    into 'build/main/resources/'
-    filter(ReplaceTokens, tokens: [projectversion: "${version}".toString()])
-}
-
+ext.bootJarPath    = relativePath(tasks.bootJar.outputs.files.singleFile)
+ext.gitShortCommit = 'git rev-parse --short=8 HEAD'.execute().text.trim()
+ext.ociImageName   = "local/${rootProject.name}"
```

Comment:
origin.url :)

Reply:
oups, sorry, I did the commit on the wrong branch this morning. It has been fixed.

Reply:
Perfect now thanks!